curriculum/challenges/english/25-front-end-development/lecture-understanding-performance-in-web-applications/67d2f7a69809807c59a44002.md

```diff
@@ -2,13 +2,39 @@
 id: 67d2f7a69809807c59a44002
 title: What Are Some Common Performance Measurement Tools?
 challengeType: 11
-videoId: nVAaxZ34khk
+videoId: sFj8e9ilwzM
 dashedName: what-are-some-common-performance-measurement-tools
 ---
 
 # --description--
 
-Watch the lecture video and answer the questions below.
+Watch the video or read the transcript and answer the questions below.
+
+# --transcript--
+
+What are some common performance measurement tools?
+
+If you want to make your website faster and more responsive, you need the right tools to measure and optimize performance. Let's explore some common performance measurement tools and how they help.
+
+First up, we have Chrome DevTools.
+
+Chrome DevTools is a built-in tool inside Google Chrome that lets you analyze and debug performance in real-time. DevTools will show loading times, CPU usage, and render delays. It's especially useful for measuring First Contentful Paint, or FCP, which we know by now, is how fast a user sees the first visible content. If your website feels slow, DevTools will help you spot the bottlenecks.
+
+Next up, Lighthouse.
+
+Lighthouse is an automated tool that checks performance, SEO, and accessibility. To use it, simply open Chrome DevTools and go to the "Lighthouse" tab. Next, click "Analyze page load" and it will give you a score on how fast and optimized your site is, as well as provide recommendations to improve it. If you want a quick performance audit, Lighthouse is your go-to tool.
```

Comment:
At 0:55 Ania says "generate report" instead of "analyze page load", which seems to be correct. Another thing we might want to fix.

Reply:
I put "Analyze page load" because that's what the button says in the video - I assume that's what you meant is correct. We could create an issue about this one.